charts/ingress-nginx/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: ingress-nginx
 # When the version is modified, make sure the artifacthub.io/changes list is updated
 # Also update CHANGELOG.md
-version: 3.34.0
+version: 3.35.0
```

Comment:
Please don't bump it here, (don't change) as we are now going to release helm charts in a batch manner.

Thanks

Reply:
OK, I also reverted the CHANGELOG update then.